src/test/java/at/ac/tuwien/kr/alpha/solver/AbstractSolverTests.java

```diff
@@ -71,17 +77,16 @@ protected static void enableDebugLog() {
 	}
 
 	/**
-	 * Calling this method in a test leads to the test being ignored for the naive solver.
-	 * Note: use this sparingly and only on tests that require too much run time with the naive solver.
+	 * Calling this method in a test leads to the test being ignored for the naive solver. Note: use this sparingly and only on tests that require too much run
+	 * time with the naive solver.
 	 */
 	void ignoreTestForNaiveSolver() {
 		org.junit.Assume.assumeFalse(solverName.equals("naive"));
 	}
 
 	/**
-	 * Calling this method in a test leads to the test being ignored for non-default domain-independent heuristics.
-	 * Note: use this sparingly and only on tests that require too much run time with non-default heuristics
-	 * (which are not tuned for good performance as well as VSIDS).
+	 * Calling this method in a test leads to the test being ignored for non-default domain-independent heuristics. Note: use this sparingly and only on tests
```

Comment:
I think the old formatting was done on purpose.

Reply:
fixed